package.json

```diff
@@ -47,13 +47,14 @@
         "node ./scripts/lint-staged/tslint",
         "git add"
       ],
-      "*.js": [
+      "*.{js,jsx,json,scss,html,yml}": [
```

Comment:
markdown files?

Reply:
Initially I was worried about including markdown files because I think it will introduce a big delta (mainly since prettier adds an extra line below headings and we don't always do that, including in changelogs). Though for the changelogs in particular that's solved by excluding them from formatting.

Reply:
I also want to try Kevin's suggestion of running prettier everywhere to see what changes before checking in. (Though I think that script might be broken......)

Reply:
Added markdown files to the list